extra/resources/attribute

```diff
@@ -233,6 +239,7 @@ migrate_to)     stop ;;
 migrate_from)   start ;;
 reload)         start ;;
 validate-all)   validate ;;
+reload-agent)   attribute_reload_agent;;
```

Comment:
We can skip implementing reload-agent for this one since there are no parameters that would be useful to make reloadable